### ISSUE TITLE
Loss >1 

### ISSUE THREAD

**Issue by AliaMYH** [fchollet/keras#5711]
How do I interpret a loss that's greater than 1? I have 10 classes that I'm classifying.

**Comment by bstriner** [comment]
Different losses have different scales, some can be negative, some can be > 1, etc. I assume you are using categorical_crossentropy? 

The loss is the cross entropy, which is a log of a probability, and is a kind of energy. Interpret it as the energy of the difference between the real distribution and the model.

The cross entropy has really good properties for finding a good fit, but isn't really something you want to try to interpret.

If you want something you can actually interpret, add `metrics=['accuracy']` to your model. A metric like 'accuracy' is a lot easier to interpret. There are plenty of other metrics you could try or you could write something custom.

Cheers
